tools/windows_buildenv.bat

```diff
@@ -65,7 +65,12 @@ EXIT /B 0
             REM TODO: verify download using sha256sum?
         )
         ECHO ### Unpacking !BUILDENV_PATH!.zip ###
-        CALL :UNZIP "!BUILDENV_PATH!.zip" "!BUILDENV_BASEPATH!"
+        CALL :DETECT_7ZIP
+        IF NOT DEFINED RETVAL (
+            CALL :UNZIP "!UNZIP_POWERSHELL!.zip" "!BUILDENV_BASEPATH!"
```

Comment:
@JosepMaJAZ This looks weird, are you sure this is correct? it looks like it should say:

```suggestion
            CALL :UNZIP_POWERSHELL "!BUILDENV_PATH!.zip" "!BUILDENV_BASEPATH!"
````